.github/workflows/kaakaww.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+name: KAAKAWW
```

Comment:
@HCloward @kcberg this is the filename that Scott uses for his demos, thinking we should rename it so it doesn't collide or break any existing demos.